internal/reflect.go

```diff
@@ -31,34 +31,44 @@ func init() {
 	interfaceTyp = reflect.TypeOf(&x).Elem()
 }
 
-// ToArray converts a []interface{} slice into an equivalent fixed-length array
+// MakeHashable converts a []interface{} slice into an equivalent fixed-length array
```

Comment:
nit: consider doing the rename in a separate step to make the semantic change easier to read

Reply:
Done